tests/src/Unit/Plugin/Tamper/ExplodeTest.php

```diff
@@ -13,22 +13,54 @@ class ExplodeTest extends UnitTestCase {
   /**
    * Test explode.
    */
-  public function testExplode() {
-    $config = [];
-    $plugin = new Explode($config, 'explode', []);
-    $this->assertArrayEquals(['foo', 'bar', 'baz', 'zip'], $plugin->tamper('foo,bar,baz,zip'));
+  public function testExplodeWithSingleValue() {
+    $original = 'foo,bar,baz,zip';
+    $expected = ['foo', 'bar', 'baz', 'zip'];
+    $this->assertArrayEquals($expected, $this->getPluginDefaultConfig()->tamper($original));
+  }
+
+  /**
+   * Test explode.
+   */
+  public function testExplodeWithMultipleValues() {
+    $original = ['foo,bar', 'baz,zip'];
+    $expected = [['foo', 'bar'], ['baz', 'zip']];
```

Comment:
Again, no idea if we should expect this or ['foo', 'bar', 'baz', 'zip']

Reply:
I think what is in 'expected' now, makes more sense.

So exploding `['foo,bar', 'baz,zip']` by a comma should result into `[['foo', 'bar'], ['baz', 'zip']]`. If someone wants a singular array, we could port Migrate's Flatten process plugin to Tamper.